examples/container-service-extension/v4.1/install/step2/3.11-cse-install-6-ovas.tf

```diff
@@ -0,0 +1,75 @@
+# ------------------------------------------------------------------------------------------------------------
+# CSE v4.1 installation:
+#
+# * Please read the guide present at https://registry.terraform.io/providers/vmware/vcd/latest/docs/guides/container_service_extension_install
+#   before applying this configuration.
+#
+# * Rename "terraform.tfvars.example" to "terraform.tfvars" and adapt the values to your needs.
+#   Other than that, this snippet should be applied as it is.
+#   You can check the comments on each resource/data source for more help and context.
+# ------------------------------------------------------------------------------------------------------------
+
+# In this section we create two Catalogs, one to host all CSE Server OVAs and another one to host TKGm OVAs.
+# They are created in the Solutions organization and only the TKGm will be shared as read-only. This will guarantee
+# that only CSE admins can manage OVAs.
+resource "vcd_catalog" "cse_catalog" {
```

Comment:
In other HCLs you have these comments above each resource:

```
# This should not be changed.
```

I like this comment, as it leaves no doubt that the configuration shall not be changed.

Is there a reason not to have them in this file as well?
Also, there are some other places were this comment is missing, but maybe should be there. Would you double check?

Reply:
Some resources are required "as-is" because CSE requires them as they are in the proposed sample configuration. On the other hand, Orgs, VDCs, Catalogs, etc, allow more customisation in users' side (for example, they could even have the catalog already configured and this HCL block would not be needed), they can change many aspects that match with their needs.

Yep, let me double check!

Reply:
Done, I think all are good.